src/SwapExtractionTool/DistributedSwapTransaction.cs

```diff
@@ -2,7 +2,7 @@
 {
     public sealed class DistributedSwapTransaction
     {
-        public int BlockHeight { get; set; }
+        public long BlockHeight { get; set; }
```

Comment:
We have about 7 times more instances of `int` block heights than `long`. `int` has a maximum value of `2,147,483,647`.

Reply:
Sorry this change should be reverted.